### ISSUE TITLE
3.11.3 release build failed to build against go1.20

### ISSUE THREAD

**Issue by chenrui333** [hairyhenderson/gomplate#1614]
👋 while [regression testing go1.20 upgrade](https://github.com/Homebrew/homebrew-core/pull/122082), I ran into the following error:

```
==> /opt/homebrew/Cellar/gomplate/3.11.3/bin/gomplate --version
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.20 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.20 set. Notably, if go1.20 adds a moving garbage collector, this program is unsafe to use.

goroutine 1 [running]:
go4.org/unsafe/assume-no-moving-gc.init.0()
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/go4.org/unsafe/assume-no-moving-gc@v0.0.0-20220617031537-928513b29760/untested.go:25 +0x178
```

full build log, https://pipelines.actions.githubusercontent.com/serviceHosts/d065b819-d194-4baa-bc84-7b4a294d8693/_apis/pipelines/1/runs/564571/signedlogcontent/19?urlExpires=2023-02-03T17%3A06%3A38.0355252Z&urlSigningMethod=HMACV1&urlSignature=pSX%2FY%2FAHvb9NFrLSm1xQHLgwZ%2FwQ5NinBMs1z6lh5sI%3D

**Comment by hairyhenderson** [comment]
@chenrui333 yeah - gomplate 3.11.3 is definitely not Go 1.20 compatible.

It should (in theory?) work from `main`, but I'm not ready to cut a new release yet.

Can we keep this on Go 1.19 for now?

**Comment by chenrui333** [comment]
yeah, totally, raising the issue mostly for awareness. :)

**Comment by hairyhenderson** [comment]
ok, thanks! 🙇‍♂️

**Comment by hairyhenderson** [comment]
Fixed in #1641
